exercises/concept/black-jack/.docs/instructions.md

```diff
@@ -1,49 +1,85 @@
-In this exercise, you're going to implement some rules from [Blackjack][blackjack]
+# Instructions
 
-You have some rules to implement for judging the result of the game.
+In this exercise you are going to implement some rules of [Blackjack][blackjack],
+such as the way the game is played and scored.
 
-**Note** : In this exercise, _A_ means an ace, _J_ means jack, _Q_ means queen, and _K_ means king card.
+**Note** : In this exercise, _A_ means ace, _J_ means jack, _Q_ means queen, and _K_ means king cards.
 
-## 1. Calculate the number of card
+## 1. Calculate the value of a card
 
-Create the `number_of_card()` function with a parameter `card`. The value of _J, Q_ or _K_ is 10. If the `card` is _A_, then just return "ace".
+In Blackjack, it is up to each individual player if an ace is worth 1 or 11 points.
+Face cards are worth 10 points and any other card is worth its pip value.
+
+Define the `value_of_card` function with a parameter `card`.
+The value of _J_, _Q_ or _K_ is 10.
+Otherwise, return the pip value of a card.
```

Comment:
Again -- here we use "pip value", but if there is a chance that will be misunderstood, we should put in a link.  I found this one:  https://ourpastimes.com/pip-playing-cards-4616754.html, but there might be better ones out there.

Reply:
Same as above -- added clarification in parentheses and a Wikipedia link (the same about the standard 52-card deck) in the opening note.